src/Utils/versioninfo.jl

```diff
@@ -4,7 +4,7 @@ using Oceananigans.Architectures
 
 function versioninfo_with_gpu()
     s = sprint(versioninfo)
-    if CUDA.has_cuda()
+    if isdefined(Main, :CUDABackend)
         gpu_name = CUDA.CuDevice(0) |> CUDA.name
```

Comment:
I suspect this will fail, since CUDA is not defined.

Solving this is tricky, but most likely you will want to define a callback that the extension registers.

Reply:
We should change how this function works; it should accept the architecture. Right now it is only used for NetCDF output writing

Reply:
Happy to fix this, let me know what to do